spec/core/data_structures.md

```diff
@@ -278,297 +308,53 @@ Valid Light Client Attack Evidence encompasses three types of attack and must ad
 
 - Evidence must not have expired. The height (and thus the time) is taken from the common height.
 
```

Comment:
The issue about being "expired" is the same as above. I am not sure it belongs to a document about data structures.  
Also, all the other rules implicitly assume validation at a full node against a local copy of the blockchain (e.g., "invalid" and "conflicting header" in bullets 1, 2 and 3). This should be made clear.